### ISSUE TITLE
module app specified twice on command line

### ISSUE THREAD

**Issue by JesseKPhillips** [dlang/dub#1407]
### System information
- **dub version**: DUB version 1.8.0, built on Mar  2 2018
- **OS Platform and distribution**: Windows 8
- **compiler version** DMD 2.079

### Bug Description
Receiving the following error when trying to build docs through dub:
Error: module `app` from file ..\packages\ddox-0.16.8\ddox\source\app.d is specified twice on the command line

### How to reproduce?
Run> dub build --build ddox

**Comment by LaurentTreguier** [comment]
I am experiencing the same problem with dcd 0.9.1, on Windows 10 x64 with dub 1.8.0 and DMD 2.079.0 :
```
>dub build dcd --config=server
Error: module `server.server` from file ..\..\..\..\AppData\Local\dub\packages\dcd-0.9.1\dcd\src\server\server.d is specified twice on the command line
```

**Comment by s-ludwig** [comment]
Fixed by #1410.
